automated install/basic-install.sh

```diff
@@ -1357,27 +1424,61 @@ check_service_active() {
   fi
 }
 
-# Systemd-resolved's DNSStubListener and dnsmasq can't share port 53. 
+# Systemd-resolved's DNSStubListener and dnsmasq can't share port 53.
 disable_resolved_stublistener() {
   echo -en "  ${INFO} Testing if systemd-resolved is enabled"
-  # Check if Systemd-resolved's DNSStubListener is enabled and active on port 53
-  if check_service_active "systemd-resolved"; then
-    # Check if DNSStubListener is enabled
-    echo -en "  ${OVER}  ${INFO} Testing if systemd-resolved DNSStub-Listener is active"
-    if ( grep -E '#?DNSStubListener=yes' /etc/systemd/resolved.conf &> /dev/null ); then
-      # Disable the DNSStubListener to unbind it from port 53
-      # Note that this breaks dns functionality on host until dnsmasq/ftl are up and running
-      echo -en "${OVER}  ${TICK} Disabling systemd-resolved DNSStubListener"
-      # Make a backup of the original /etc/systemd/resolved.conf
+  # Check if Systemd-resolved's DNSStubListener flag is present
+  if [[ $systemd_resolved_flag = "true" ]]; then
+    if check_service_active "systemd-resolved"; then
+      # Check if DNSStubListener is enabled
+      echo -e "  ${INFO} Testing if systemd-resolved DNSStub-Listener is active"
+      if ( grep -E '#?DNSStubListener=yes' /etc/systemd/resolved.conf &> /dev/null ); then
+        # Disable the DNSStubListener to unbind it from port 53
+        # Note that this breaks dns functionality on host until dnsmasq/ftl are up and running
+        echo -e "  ${TICK} Disabling systemd-resolved DNSStubListener"
+        # Make a backup of the original /etc/systemd/resolved.conf
+        # (This will need to be restored on uninstallation)
+        ${SUDO} sed -r -i.orig 's/#?DNSStubListener=yes/DNSStubListener=no/g' /etc/systemd/resolved.conf
+        echo -e "${TICK}  Restarting systemd-resolved DNSStubListener"
```

Comment:
This needs the two space indentation in the echo, and only one space between `TICK` and the message.